index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width" , initial-scale=1.0>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" href="http://fonts.googleapis.com/css?family=Roboto">
+</head>
+
+<body>
+    <header>
+        <a href="/index.html">Ashmal Zahra </a>
+        <button type="menu" class="menu">
+            <img src="images/Union.png" alt="Menu">
+        </button>
```

Comment:
I believe the `<nav>` tag was correct before. You just needed to wrap the image around the `<button>` tag. Kindly follow the steps below to make it semantically correct 👍🏽 

- [ ] Please consider wrapping the menu button around the `<nav></nav>` tag for semantic and UX purposes. It will segregate your code in a clean and nice way.  🙂 

```
<nav>
   <button type="menu" class="menu">
       <img src="images/Union.png" alt="Menu">
   </button>
</nav>